superset/assets/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "tdd": "jest --watch",
     "test": "jest",
-    "cover": "jest --maxWorkers=8 --coverage",
+    "cover": "jest --coverage",
```

Comment:
unrelated to this PR, but this seems to help quite a bit with broken JS builds. For reference, not specifying `maxWorkers` will lead to use all CPUs cores on the box minus one